cmd/fetch.go

```diff
@@ -24,6 +24,11 @@ var fetchCmd = &cobra.Command{
 			return
 		}
 
+		// check if we actually have any targets to run
+		if len(targets) <= 0 {
+			log.Error().Msg("must specify a target")
```

Comment:
We probably want to exit failure here, right?

Reply:
Yes.